app/templates/components/public/ticket-list.hbs

```diff
@@ -0,0 +1,68 @@
+<div class="ui">
+  <h3 class="ui header">{{t 'Ticket Information'}}</h3>
+  <div class="ui divider"></div>
+  <table class="ui line table" style="margin-bottom: 0">
+    <thead class="full-width">
+      <tr>
+        <th>{{t 'TICKET TYPE'}}</th>
+        <th>{{t 'SALES ENDS'}}</th>
+        <th>{{t 'PRICE'}}</th>
+        <th>{{t 'QUANTITY'}}</th>
+        <th>{{t 'SUBTOTAL'}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each tickets as |ticket|}}
+        <tr>
+          <td>{{ticket.name}}</td>
+          <td>Thu, May 25</td>
+          <td id="{{ticket.id}}_price">{{format-currency ticket.price}}</td>
+          <td>
+            {{#ui-dropdown class='compact selection' forceSelection=false}}
+              {{input type='hidden' id=(concat ticket.id '_quantity') value=ticket.ticketOrder}}
+              <i class="dropdown icon"></i>
+              <div class="default text">0</div>
+              <div class="menu">
+                <div class="item" data-value="0">0</div>
```

Comment:
And @geekyd ... forgot to mention, You can use the `range` helper for the range. 
```hbs
{{#each (range 1 100) as |count|}}
  {{count}}
{{/each}}
```